utils/update_database.sh

```diff
@@ -56,6 +58,12 @@ do
 
     echo "Attempting to import diffs"
     nominatim add-data --diff ${DIR}/${FILENAME}.osc.gz
+
+    echo "Attempting to update date of db"
+    zgrep timestamp ${DIR}/${FILENAME}.osc.gz | tee newtimestamps.txt
+    NEWTIME = $(python3 parsedate.py newtimestamps.txt)
```

Comment:
You could pipe the timestamps directly into your helper program. Then you don't need to worry about the temporary file.

Also, you don't catch the case here where there is no timestamp. I think that can happen, when there is no new data for a country (happens with small ones like Andorra).